pep-0597.rst

```diff
@@ -33,135 +34,140 @@ Using the default encoding is a common mistake
 Developers using macOS or Linux may forget that the default encoding
 is not always UTF-8.
 
-For example, ``long_description = open("README.md").read()`` in
-``setup.py`` is a common mistake. Many Windows users can not install
-the package if there is at least one non-ASCII character (e.g. emoji)
-in the ``README.md`` file which is encoded in UTF-8.
+For example, using ``long_description = open("README.md").read()`` in
+``setup.py`` is a common mistake. Many Windows users cannot install
+such packages if there is at least one non-ASCII character (e.g. emoji)
```

Comment:
Another common example would be the copyright symbol; not sure if its worth adding

Reply:
In my personal experience, author/contributor names are more common source of non-ASCII characters.

Reply:
```suggestion
such packages if there is at least one non-ASCII character
(e.g. emoji, author names, copyright symbols, and the like)
```

Good point, thanks. What about this?

Reply:
LGTM

Reply:
:+1: Feel free to apply

Reply:
Okay, thanks! I'll apply once you :+1: everything (I forgot this was my PR and not yours, sorry!)